src/Service/IncentiveImportService.php

```diff
@@ -190,4 +209,157 @@ public function getAjaxData($incentiveImport, $incentiveImportRows)
         }
         return $rows;
     }
+
+    public function sendIncentive($participantId, $incentive, $user): bool
```

Comment:
Unless I missed something, it looks like this is only used from inside the `sendIncentivesToRdr` method below. Can this be changed to a private method?

Or, could the IncentiveService be used here? It looks like the method there is coupled to the form, but it might be worth having just one place in our code where we send an incentive to the RDR.